app/scripts/models/auth_brokers/fx-firstrun-v2.js

```diff
@@ -15,20 +15,101 @@ define(function (require, exports, module) {
   const _ = require('underscore');
   const Constants = require('lib/constants');
   const FxFirstrunV1AuthenticationBroker = require('./fx-firstrun-v1');
+  const NotifierMixin = require('views/mixins/notifier-mixin');
```

Comment:
This move was taken care of by @philbooth in https://github.com/mozilla/fxa-content-server/pull/4718